source/compiler/sc2.c

```diff
@@ -1082,7 +1082,7 @@ static int command(void)
     } /* if */
     check_empty(lptr);
     break;
-  case tINCLUDE:                /* #include directive */
+  case tpINCLUDE:                /* #include directive */
```

Comment:
Why has this symbol gained a `p`?

Reply:
Name consistent with others
![image](https://user-images.githubusercontent.com/1020099/50845090-2854da00-137d-11e9-8437-fb4a05556c86.png)

Reply:
It mostly stands for: `t[oken]p[reprocessor]include`